scripts/test_all.py

```diff
@@ -0,0 +1,45 @@
+import os
+import csv
+from argparse import ArgumentParser
+from .preprocess import preprocess
+from .test_single import test_single
+from .load_tools import setup_psiminer, setup_code2seq
+
+
+def test_all(dataset_path: str, model_path: str, results_path: str):
+    """Evaluate and store all"""
+
+    project_names = os.listdir(dataset_path)
+    for project_name in project_names:
+        preprocess(os.path.join(dataset_path, project_name))
+
+    project_names = os.listdir("datasets")
```

Comment:
Why do you have both “dataset_path” as an argument and “datasets” as a constant? Is it a bug? Anyway, constant can be moved to the utils file (as it is done for the dependencies).